htdocs/comm/propal.php

```diff
@@ -1217,7 +1218,12 @@
 /*
  * View
  */
-
+ 
+ // If const defined, creation = society search
+if ($action == 'create' &&	dolibarr_get_const($db, 'PROPAL_NEW_SEARCH_SOC')) {
+	header("Location: ../societe/list.php?leftmenu=thirdparties");
+	exit();
```

Comment:
This code is unreachable and useless plus exit should only be used in CLI scripts, not CGI.